packages/calcite-components/src/components/action/action.e2e.ts

```diff
@@ -144,15 +144,15 @@ describe("calcite-action", () => {
     const page = await newE2EPage();
     await page.setContent(`<calcite-action indicator text="hello world"></calcite-action>`);
 
-    const button = await page.find(`calcite-action >>> .${CSS.button}`);
+    const button = await page.find(`calcite-action >>> button`);
```

Comment:
should we go back to using the class here? It just seems like the class is less feeble. Not that the button element would ever change though. Up to you :)